sdk/openai/Azure.AI.OpenAI/src/Custom/EmbeddingsOptions.cs

```diff
@@ -13,6 +13,8 @@ namespace Azure.AI.OpenAI
     /// <summary> Schema to create a prompt completion from a deployment. </summary>
     public partial class EmbeddingsOptions
     {
+        internal string InternalNonAzureModelName { get; set; }
+
         /// <summary> Initializes a new instance of EmbeddingsOptions. </summary>
```

Comment:
I believe this constructor can be deleted from here and auto-generated.

Reply:
Awesome, indeed it can!